src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+import java.util.Scanner;
+
+public class Duke {
+
+    Scanner in = new Scanner(System.in);
+    private String commands;
+    Task[] tasks = new Task[100];
+
+    Duke() {
+        System.out.println("Hello, I am Duke!");
+        System.out.println("What can I do for you?");
+    }
+
+    void processCommands(){
+        int i = 0;
+        while(true) {
+            commands = in.nextLine();
+            String[] words = commands.split(" ");
```

Comment:
Perhaps these two lines could be refactored out as a method? It could help ensure that processCommands() adhere to SLAP more closely.
e.g. getUserInput(), splitCommand().